src/google/protobuf/io/io_win32.h

```diff
@@ -75,6 +76,26 @@ PROTOBUF_EXPORT int stat(const char* path, struct _stat* buffer);
 PROTOBUF_EXPORT int write(int fd, const void* buffer, size_t size);
 PROTOBUF_EXPORT std::wstring testonly_utf8_to_winpath(const char* path);
 
+struct ExpandWildcardsResult {
+  enum {
```

Comment:
Since we're now using C++11, you could make this an `enum class` instead of just `enum`, and that will scope the enum without the need for an enclosing struct.

Reply:
Thanks! Done.